src/gleam/erlang/process.gleam

```diff
@@ -109,8 +176,21 @@ fn raw_send(a: Pid, b: message) -> DoNotLeak
 /// ```
 ///
 pub fn send(subject: Subject(message), message: message) -> Nil {
-  raw_send(subject.owner, #(subject.tag, message))
-  Nil
+  case subject {
+    Subject(pid, tag) -> {
+      raw_send(pid, #(tag, message))
+      Nil
+    }
+    NamedSubject(name) -> {
+      case named(name) {
+        Ok(pid) -> {
+          raw_send(pid, #(name, message))
+          Nil
```

Comment:
Silently dropping for unregistered pid is fine for me, but it differs from the Erlang implementation. 
```
21> register(hej, self()).
true
22> erlang:send(hej, hoj).
hoj
23> erlang:send(hj, hoj).
** exception error: bad argument
     in function  erlang:send/2
        called as erlang:send(hj,hoj)
        *** argument 1: invalid destination
``